banzai_nres/traces.py

```diff
@@ -9,43 +9,42 @@
 import logging
 from scipy import ndimage
 import numpy as np
-from banzai_nres.fitting import fit_smooth_spline
+from numpy.polynomial.polynomial import Polynomial, polyfit
 
 logger = logging.getLogger('banzai')
 
 # Minimum separation between peaks in the image that could be separate traces
 MIN_TRACE_SEPARATION = 10
 
 # Cutoff in signal-to-noise to stop following a trace
-SIGNAL_TO_NOISE_TRACING_CUTOFF = 50
+SIGNAL_TO_NOISE_TRACING_CUTOFF = 10
 
-# Minimum half width of a feature to be considered a trace
-MIN_TRACE_HALF_WIDTH = 50
+# Maximum degree of polynomial which describes the y position of the trace as a function of pixel
```

Comment:
We should use legendre polynomials. Regular polynomials are not orthogonal. We also need to scale the inputs from -1 to 1 (for orthogonality).

Reply:
Ok. This is easy to change with the numpy Legendre module. I will do that soon.

Reply:
The x scaling is the only tricky part. We don't really want the user of the fit to have to do the rescaling. You may want to return a wrapped best fit object or something.